widget/list.go

```diff
@@ -239,6 +254,39 @@ func (l *List) ScrollToTop() {
 	l.Refresh()
 }
 
+// TypedKey is called if a key event happens while this List is focused.
+//
+// Implements: fyne.Focusable
+func (l *List) TypedKey(event *fyne.KeyEvent) {
+	switch event.Name {
+	case fyne.KeySpace:
+		l.Select(l.currentFocus)
+	case fyne.KeyDown:
+		if f := l.Length; f != nil && l.currentFocus >= f()-1 {
+			return
+		}
+		l.RefreshItem(l.currentFocus)
+		l.currentFocus++
+		l.scrollTo(l.currentFocus)
```

Comment:
Can the `scrollTo` API be made public as well (same rationale as above comment)

Reply:
That would be consistent with the other collections

Reply:
Nevermind, there's already the public ScrollTo, which just does bounds checking and then calls scrollTo

Reply:
Erm... as far as I can see List already has a public ScrollTo